### PR TITLE
Sorting options are incomplete

### LINKED ISSUE #1190: Sorting options are incomplete
## Issue 
Sorting options are not explicit as they don't indicate the direction of sorting, e.g. ascending and descending. 
Screen Shot 2020-04-13 at 11 51 33 AM

✅ The following sorting options available:

- Newest first (default sorting) 
- Oldest first 
- Directory: Descending 
- Directory: Ascending 
- Name: A-Z
- Name: Z-A

### PR DESCRIPTION
**Pull request by engcom-Charlie**
### Fixed Issues (if relevant)

1. magento/adobe-stock-integration#1190: Sorting options are incomplete
2. ...

### Manual testing scenarios (*)

1. ...
2. ...

### CODE REVIEW

**Comment by engcom-Charlie** [comment]
@magento run Functional Tests EE

File: MediaGalleryUi/view/adminhtml/web/js/grid/sortBy.js [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+        /**
+         * Prepared sort order options
+         */
+        preparedOptions: function (columns) {
+            var index = 0;
+
+            if (columns && columns.length > 0) {
+                columns.map(function (column, i) {
+                    if (column.sortable === true) {
+                        this.options.push({
```
Comment (sivaschenko) [inline MediaGalleryUi/view/adminhtml/web/js/grid/sortBy.js:26]: Is it possible to provide these options from XML configuration of the sorting component?

**Review by sivaschenko — CHANGES_REQUESTED** [review]
Thanks for the pull request @engcom-Charlie ! Please see my review comments

**Review by VladimirZaets — DISMISSED** [review]

**Review by sivaschenko — CHANGES_REQUESTED** [review]
Thanks for the PR @engcom-Charlie , can you please fix the static tests

**Review by chalov-anton — CHANGES_REQUESTED** [review]
The default _Sort by_ is empty 
sort_by_empty
Please, change it to _Newest First_ to be the default

**Comment by engcom-Charlie** [comment]
@chalov-anton Please check this issue now.

**Comment by chalov-anton** [comment]
:heavy_check_mark: QA Passed

**Comment by lenaorobei** [comment]
@magento run Functional Tests CE, Functional Tests B2B

**Comment by lenaorobei** [comment]
https://m2build-ur.devops.magento.com/view/all/job/All-User-Requested-Tests/31818/

**Comment by lenaorobei** [comment]
@magento run all tests

**Comment by lenaorobei** [comment]
@magento run Functional Tests B2B, Functional Tests CE, Unit Tests

**Comment by lenaorobei** [comment]
@magento run Unit Tests

**Pull request merged**